### DISCUSSION TITLE
Intel Movidius TPU

### DISCUSSION THREAD

**Discussion by RichardPar** [blakeblackshear/frigate#1109] [Ask A Question]
Hi there,

Has there been any experiments using an Intel Movidius (Neural Compute Stick 2)  ? It uses OpenCV + OpenVINO, no idea how it would integrate with Frigate.

Regards,
Richard

**Comment by blakeblackshear** [comment]
I have never tried anything with this TPU. The Coral is cheaper and faster.

**Comment by RichardPar** [comment]
Almost agreed,  Coral is hard to find in the UK .. where they are in stock,
they are more expensive than Intel :D

Richard

**Comment by blakeblackshear** [comment]
That should be temporary, and it won't be trivial to support the NCS.

**Reply by magicmellon** [reply]
Hi @blakeblackshear,

Avaliability has unfortunately not eased up in the last 5 months. I have found a listing for some £50 Intel Neural Compute Stick 2's but I assume from this comment it would not be just plug and play.

Would you be able to give me any information on how you began integrating the google corral stick into frigate? Then I will investigate integrating it myself!

Thanks for all the work you do on this project - the latest updates UI changes are awesome :)

**Reply by akston** [reply]
@blakeblackshear can you please address this question regarding Intel NCS integration? Availability for the Google Coral still hasn't improved.

**Reply by blakeblackshear** [reply]
It will require major changes to support. New drivers in the container, significant changes to the model prep, a different model, and a new object detection process type.

**Comment by fcasarsa** [comment]
I have quite annoying false positives, maybe these list of pre-trained models have a better accuracy than Coral.

https://docs.openvinotoolkit.org/latest/omz_models_model_person_detection_0203.html

Is there a way to do a second step inferencing calling an external bash script?

EDIT:

look at this example, https://docs.openvinotoolkit.org/latest/omz_demos_crossroad_camera_demo_cpp.html


Thanks.
